### DISCUSSION TITLE
UDP packets from container to Wyze Cam V3 during streaming...why?

### DISCUSSION THREAD

**Discussion by AdiAbuAli** [mrlt8/docker-wyze-bridge#263] [Q&A]
Hi,
I did changed the stream transport to UDP, and took tcpdump from container as -i , and I can see some UDP packets sent from containers back to cam....what normally those packets for? sync? they all have same size.

[docker0.zip](https://github.com/mrlt8/docker-wyze-bridge/files/7636146/docker0.zip)

10.0.0.2, 10.0.0.16, 10.0.023 are wyze cam IPs

172.17.0.3 is the Wyze bridge container
172.17.0.1 is the NVR (in same platform)

Attached is the PCAP.

================================================================================================================================================
IPv4 Statistics/Source and Destination Addresses:
Topic / Item                     Count         Average       Min Val       Max Val       Rate (ms)     Percent       Burst Rate    Burst Start  
------------------------------------------------------------------------------------------------------------------------------------------------
Source IPv4 Addresses            29149                                                   0.6724        100%          3.8200        29.801       
 172.17.0.3                      11278                                                   0.2601        38.69%        3.6200        29.807       
 172.17.0.1                      12                                                      0.0003        0.04%         0.0400        13.921       
 10.0.0.23                       5248                                                    0.1211        18.00%        1.4800        42.840       
 10.0.0.2                        6057                                                    0.1397        20.78%        1.3000        35.916       
 10.0.0.16                       6554                                                    0.1512        22.48%        1.2500        23.146       
Destination IPv4 Addresses       29149                                                   0.6724        100%          3.8200        29.801       
 172.17.0.3                      17871                                                   0.4122        61.31%        1.9200        21.085       
 172.17.0.1                      6395                                                    0.1475        21.94%        3.4700        29.826       
 10.0.0.23                       **1496**                                                    0.0345        5.13%         0.2100        36.094       
 10.0.0.2                        **1680**                                                    0.0388        5.76%         0.2200        35.928       
 10.0.0.16                       **1707**                                                    0.0394        5.86%         0.0900        34.652       

------------------------------------------------------------------------------------------------------------------------------------------------

**Comment by mrlt8** [comment]
The tutk library uses UDP to communicate with the camera.
